GoRogue.Debugger/Extensions/MapExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using GoRogue.GameFramework;
+using GoRogue.MapViews;
+
+namespace GoRogue.Debugger.Extensions
+{
+    public static class MapExtensions
+    {
+        public static ArrayMap<char> CharMap(this Map map)
+        {
+            ArrayMap<char> chars = new ArrayMap<char>(map.Width, map.Height);
```

Comment:
I feel like a lambda map view might be a nicer approach here as it would reduce some lines of code and reduce the allocation, but this works functionally as well.